### PR TITLE
Charlotte

### PR DESCRIPTION
**Pull request by jialeCharloote**

### CODE REVIEW

File: .DS_Store
Comment (QinPR) [inline .DS_Store:1]: Hi Charlotte, please avoid committing this file. This is a junk file that can trash our repo. You may find `.gitignore` useful to ignore committing this file.

File: .osm_rd_congo_localities.geojson.icloud
Comment (QinPR) [inline .osm_rd_congo_localities.geojson.icloud:1]: Please also remove this file or back up it somewhere else. I think this file is for Congo and not related to Ghana.

**Review by QinPR — CHANGES_REQUESTED** [review]
The following code are our old-version code about congo, please don't merge them into the main branch in the current pull request:
1. Mining_feature.ipynb
2. all_mining_location.xlsx
3. local_roads
4. main_roads
5. protected_areas

File: .DS_Store
Comment (jialeCharloote) [reply .DS_Store:1]: gotcha thx

**Pull request merged**